README.md

```diff
@@ -1,22 +1,60 @@
+  <a href="logos/BookBrainz/"> <img src="logos/All%20Logos/BookBrainz.svg" alt="BookBrainz" title="BookBrainz" width="19%"></a>  
+  <a href="logos/AcousticBrainz/"> <img src="logos/All%20Logos/AccousticBrainz.svg" alt="AcousticBrainz" title="AcousticBrainz" width="19%"></a>  
+  <a href="logos/CritiqueBrainz/"> <img src="logos/All%20Logos/CritiqueBrainz.svg" alt="CritiqueBrainz" title="CritiqueBrainz" width="19%"></a>  
+  <a href="logos/Picard/"> <img src="logos/All%20Logos/MusicBrainz%20Picard.svg" alt="Picard" title="Picard" width="19%"></a>  
+  <a href="logos/Cover Art Archive/"> <img src="logos/All%20Logos/CoverArtArchive.svg" alt="CoverArtArchive" title="CoverArtArchive" width="19%"></a>  
+  <a href="logos/MessyBrainz/"> <img src="logos/All%20Logos/MessyBrainz.svg" alt="MessyBrainz" title="MessyBrainz" width="19%"></a>  
+  <a href="logos/MetaBrainz Community/"> <img src="logos/All%20Logos/MetaBrainz%20Community.svg" alt="MetaBrainz Community" title="MetaBrainz Community" width="19%"></a>  
 </div>
 
-<div>
-  <img src="logos/All%20Logos/MusicBrainz.svg" alt="MusicBrainz" title="MusicBrainz" width="198">
-  <img src="logos/All%20Logos/ListenBrainz.svg" alt="ListenBrainz" title="ListenBrainz" width="198">
-  <img src="logos/All%20Logos/BookBrainz.svg" alt="BookBrainz" title="BookBrainz" width="198">
-  <img src="logos/All%20Logos/AccousticBrainz.svg" alt="AcousticBrainz" title="AcousticBrainz" width="198">
-  <img src="logos/All%20Logos/CritiqueBrainz.svg" alt="CritiqueBrainz" title="CritiqueBrainz" width="198">
-  <img src="logos/All%20Logos/MusicBrainz%20Picard.svg" alt="Picard" title="Picard" width="198">
-  <img src="logos/All%20Logos/CoverArtArchive.svg" alt="CoverArtArchive" title="CoverArtArchive" width="198">
-  <img src="logos/All%20Logos/MessyBrainz.svg" alt="MessyBrainz" title="MessyBrainz" width="198">
-  <img src="logos/All%20Logos/MetaBrainz%20Community.svg" alt="MetaBrainz Community" title="MetaBrainz Community" width="198">
-</div>
+# Guidelines
+
+## Sizes
+Use logo sizings consistently across projects. For instance, *Brainz logo mini* is used as the top left logo across all MetaBrainz projects. When unsure, refer to prior projects and useage.</br>
+Although the icons should never be smaller than the minimum sizes\*, you may increase sizes as you wish. For instance, the usually small *Brainz icon* can be used in a very large format, if required.
+
+><img src="logos/MetaBrainz/SVG/MetaBrainz_logo_icon.svg" alt="MetaBrainz icon" height="24px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_mini.svg" alt="MetaBrainz logo mini" height="23px"></br>
+>*Brainz icon / Brainz logo mini*</br>
+>No minimum
+
+><img src="icons/metabrainz-icon-detail.svg" alt="MetaBrainz icon detail" height="34px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_small.svg" alt="MetaBrainz logo small" height="33px"></br>
+>*Brainz icon detail / Brainz logo small*</br>
+>Minimum height: 33px/12mm</br>
```

Comment:
Not sure if it is relevant, but on desktop a typical icon size is 32px, and for Picard we start using this icon at that size.

Reply:
Wow, pretty close though! Thanks, I've dropped the minimum to 32px to reflect what was tested and worked. I left the print (mm) minimum the same.